addons/hr_skills_survey/models/survey_user.py

```diff
@@ -19,16 +20,29 @@ def _mark_done(self):
         certification_user_inputs = self.filtered(lambda user_input: user_input.survey_id.certification and user_input.scoring_success)
         partner_has_completed = {user_input.partner_id.id: user_input.survey_id for user_input in certification_user_inputs}
         employees = self.env['hr.employee'].sudo().search([('user_id.partner_id', 'in', certification_user_inputs.mapped('partner_id').ids)])
+        resume_surveys = self.env['hr.resume.line'].search(
+            expression.AND([
+                expression.OR([[('employee_id', '=', employee.id)],
+                               [('survey_id', '=', partner_has_completed[employee.user_id.partner_id.id].id)]])
+                for employee in employees
+            ]))
+        resume_survey_by_ids = {(resume_survey.employee_id, resume_survey.survey_id): resume_survey
+                                for resume_survey in resume_surveys}
         for employee in employees:
             line_type = self.env.ref('hr_skills_survey.resume_type_certification', raise_if_not_found=False)
-            survey = partner_has_completed.get(employee.user_id.partner_id.id)
-            self.env['hr.resume.line'].create({
+            survey = partner_has_completed[employee.user_id.partner_id.id]
+            vals = {
                 'employee_id': employee.id,
                 'name': survey.title,
                 'date_start': fields.Date.today(),
-                'date_end': fields.Date.today(),
-                'description': html2plaintext(survey.description),
+                'date_end': False,
+                'description': not is_html_empty(survey.description) and html2plaintext(survey.description),
```

Comment:
> Actually, “html2plaintext(False)” gives 'False' (a string)

Ah, I guess it's because of that line `html = ustr(html)`, instead shouldn't we return "" in `html2plaintext`if the input is Falsy ?

there's already that, so I think it should return an empty string
```py
if not html.strip():
    return ''
```